index.js

```diff
@@ -6,30 +6,16 @@ var xtend = require("xtend")
 
 module.exports = createXHR
 createXHR.XMLHttpRequest = window.XMLHttpRequest || noop
-createXHR.XDomainRequest = "withCredentials" in (new createXHR.XMLHttpRequest()) ? createXHR.XMLHttpRequest : window.XDomainRequest
 createXHR.qsSerialize = null // Define this as a function to support the `qs` option
 
-forEachArray(["get", "put", "post", "patch", "head", "delete"], function(method) {
+"get,put,post,patch,head,delete".split(",").map(function(method) {
```

Comment:
It saves 6 characters :)
I'm generally avoiding using my old golfing tricks in real work, but this one seemed readable.

If it raises eyebrows, contrary to what I assumed, I sure have to change it.